Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:0.10-onbuild
+# replace this with your application's default port
+EXPOSE 8080 
```

Comment:
Not for Node.js, no. You can look at the `ONBUILD` instructions in the [upstream Dockerfile](https://github.com/joyent/docker-node/blob/0c2ff5172aabc30ce38303d9bb340ae3e94f3a91/0.8/onbuild/Dockerfile#L6-L8) to see what the -onbuild tagged parent container is doing for you automatically, and adapt accordingly.